policyengine_canada/parameters/gov/provinces/on/tax/income/credits/noec/age_eligibility.yaml

```
@@ -0,0 +1,9 @@
+description: Northern ontario energy credit age eligibility.
```

And all other ontarios
```suggestion
description: Northern Ontario energy credit age eligibility.
```